packages/lib/package.json

```diff
@@ -37,6 +37,9 @@
     "test:unit:coverage": "vitest run --coverage",
     "test:unit:watch": "vitest watch"
   },
+  "dependencies": {
+    "html2canvas": "1.4.1"
+  },
```

Comment:
This must go into the `devDepenencies` and `peerDependencies` since we don't want to bundle it to the `lib` and want the consumer `app` to provide it. Important: you need to also put the package inside the `rollup.config.js` array under `external`.